### DISCUSSION TITLE
How to use defineProps in nuxt-bridge

### DISCUSSION THREAD

**Discussion by 305810827** [nuxt/nuxt#18338] [Questions]
![image](https://user-images.githubusercontent.com/37141782/137096686-acf8d24a-f254-4ab4-81cd-55b93244cc10.png)
```
<script setup>
const props = defineProps({
  title : String
})
</script>

```

How to introduce defineProps? I use Nuxt Bridge now

**Comment by lucpotage** [comment]
It’s hard to answer with no code to review.

**Reply by 305810827** [reply]
I fixed the problem.

**Reply by kissu** [reply]
@305810827 Feel free to write down your answer in case somebody will be in the same situation! 👍🏼

**Comment by lucpotage** [comment]
Could you mark the question as answered?
